guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc

```diff
@@ -0,0 +1,82 @@
+[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to a syncabe format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead, on the importing {ProjectServer} you must:
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}
+* Update your CDN configuration to *Custom CDN*
+* Set the CDN URL to point to the web server
+* Optionally set a SSL CA Credential if the web server requires it
+* Enable the Repository
+* Synchronize the Repository.
+endif::[]
```

Comment:
Consider this a non-blocker, but I think we're under-selling the feature here. We should at least have a module that says "this is how you create your custom CDN before you configure Satellite Server to sync the content from it".

Reply:
I'll look into it once we will have "Syncable Imports" available in Sat.